src/components/Featured/Featured.scss

```diff
@@ -124,9 +125,39 @@ html[data-theme="light"] {
 }
 
 .featured-card__products-container {
+  max-height: 250px;
+  padding-top: 1.35rem;
+  padding-right: 1rem;
   display: grid;
   grid-template-columns: 1fr 1fr;
   grid-gap: 1rem;
+  overflow-y: auto;
+  
+  --mask-size-content: calc(100% - var(--ifm-scrollbar-size)) 100%;
+  --mask-image-scrollbar: linear-gradient(black, black);
+  --mask-size-scrollbar: var(--ifm-scrollbar-size) 100%;
+  --mask-height: 32px;
+
+  --mask-image-content: linear-gradient(
+     to bottom,
+     transparent,
+     black var(--mask-height),
+     black calc(100% - var(--mask-height)),
+     transparent
+  );
+
+  mask-image: var(--mask-image-content), var(--mask-image-scrollbar);
+  mask-size: var(--mask-size-content), var(--mask-size-scrollbar);
+  mask-position: 0 0, 100% 0;
+  mask-repeat: no-repeat, no-repeat;
+
+  &::-webkit-scrollbar { width: var(--ifm-scrollbar-size); }
+  &::--webkit-scrollbar-track { background-color: transparent; }
+  
+  &::-webkit-scrollbar-thumb {
+    border-radius: 1rem;
+    background-color: var(--ifm-scrollbar-thumb-background-color);
+  }
```

Comment:
Not mega keen on the scrollbars for the cards on the home page TBH, but that is presumably a decision outside of this specific IaC/SDK content work?

Reply:
@blindaa121 can speak more to this